src/lib/string.ts

```diff
@@ -64,3 +64,20 @@ export function truncate(value: string | undefined, maxLength: number, suffix =
 
   return `${value.slice(0, maxLength)}${suffix}`;
 }
+
+/**
+ * Splits the string at line breaks
+ * @param str the string to split
+ * @param removeEmptyEntries the option to remove empty entries
+ * @returns the individual lines as an array
+ */
+export function splitLine(str: string, removeEmptyEntries: boolean = false): string[] {
+  if (isNullOrEmpty(str)) {
+    return [];
+  }
+
+  if (removeEmptyEntries) {
+    return str.split(/\r\n|\r|\n/).filter((line) => !isNullOrWhitespace(line));

```

Comment:
I think is better to do the actual split before the check of the removeEmptyEntries so that you don't have to duplicate the code.
something like:
```
const splitted = str.split(/\r\n|\r|\n/);
return removeEmptyEntries ? splitted..filter((line) => !isNullOrWhitespace(line)) : splitted;